expirationd.lua

```diff
@@ -949,6 +1054,12 @@ local function show_task_list_obsolete(...)
 end
 
 return {
+    cfg     = setmetatable({}, {
+        __index = cfg,
+        __newindex = function() error("Use crud.cfg{} instead", 2) end,
```

Comment:
Typo: crud -> expirationd.

Reply:
I'm not a copy-paste master =( Thank you! Fixed.